### PR TITLE
Fix Redis Service Broker Connection

### LINKED ISSUE #35: Fix Redis on remote deploy
Redis is functioning normally in local deploy, but is producing silent failures that manifest as timeouts in the remote deploy.

### PR DESCRIPTION
**Pull request by jimmoffet**
Cloud.gov enforces ssl transport for its elasticache redis service broker, but provides the wrong (non-ssl protocol) connection string uri via the vcap_services environment object.

Manually replacing `redis://...` with `rediss://...` resolves the connection timeouts when trying to reach the service.

Fixes #35

### CODE REVIEW

File: app/config.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
     # from app.cloudfoundry_config import extract_cloudfoundry_config
     # extract_cloudfoundry_config()
     vcap_services = json.loads(os.environ['VCAP_SERVICES'])
-    os.environ['REDIS_URL'] = vcap_services['aws-elasticache-redis'][0]['credentials']['uri']
+    os.environ['REDIS_URL'] = vcap_services['aws-elasticache-redis'][0]['credentials']['uri'].replace("redis", "rediss")
```
Comment (jimmoffet) [inline app/config.py:10]: This is hacky. I'm going to try to convince CG to provide something like ['ssl-uri'] in the future.

File: app/notify_client/status_api_client.py [hunk 2]
```diff
@@ -10,5 +10,9 @@ def get_status(self, *params):
     def get_count_of_live_services_and_organisations(self):
         return self.get(url='/_status/live-service-and-organisation-counts')
 
+    @cache.set('live-service-and-organisation-counts', ttl_in_seconds=3600)
+    def get_count_of_live_services_and_organisations_cached(self):
+        return self.get(url='/_status/live-service-and-organisation-counts')
+
```
Comment (jimmoffet) [inline app/notify_client/status_api_client.py:17]: this is for the redis healthcheck, if manually setting a key succeeds, we try the cache decorator

File: app/config.py (hunk 1 above)
Comment (rahearn) [reply app/config.py:10]: eh - not ideal but I'm not too worried about it. Another thing to potentially look into is if the library supports using the unmodified URL but with an `ssl` flag set to true. Or building the URL from the other data in the credentials hash.

Comment (jimmoffet) [reply app/config.py:10]: Yeah, I did try that, but unfortunately there's a limitation in the base redis-py lib that flaskredis wraps. The redis.from_url method does not accept ssl as a param the way that the client does when connecting with the traditional {host: x, port: x, pass: x} dict

**Pull request merged**